### PR TITLE
Fix bug in resizing of DialogWrapper with large error label

### PR DESCRIPTION
**Pull request by jgaillard85**
If the error label in the DialogWrapper is displayed on multiple lines,
any update of the error message that keeps it on several lines will
cause the dialog window to shrink.

### CODE REVIEW

**Comment by jgaillard85** [comment]
CLA signed

**Comment by dahlstrom-g** [comment]
No individual CLA should apply to this contribution, since it's work for hire at Google and covered by the agreement between Google and JetBrains.

@jgaillard85, you should join the [Google org](https://github.com/google).

File: platform/platform-api/src/com/intellij/openapi/ui/DialogWrapper.java [hunk 1]
```diff
@@ -1253,7 +1253,7 @@ public void componentResized(ComponentEvent event) {
         if (height != myHeight) {
           myHeight = height;
           myResizeInProgress = true;
-          myErrorText.setMinimumSize(new Dimension(0, height));
+          myErrorText.setMinimumHeight(height);
```
Comment (kkmic) [inline platform/platform-api/src/com/intellij/openapi/ui/DialogWrapper.java]: This change is redundant. It technically changes nothing.

File: platform/platform-api/src/com/intellij/openapi/ui/DialogWrapper.java [hunk 2]
```diff
@@ -236,7 +236,7 @@ public void componentResized(ComponentEvent e) {
           if (!myResizeInProgress) {
             myActualSize = myPeer.getSize();
             if (myErrorText != null && myErrorText.isVisible()) {
-              myActualSize.height -= myErrorText.myLabel.getHeight();
+              myActualSize.height -= myErrorText.getMinimumHeight();
```
Comment (kkmic) [inline platform/platform-api/src/com/intellij/openapi/ui/DialogWrapper.java]: I'd put it myErrorText.getMinimumSize().height without adding new methods.

Comment (jgaillard85) [reply platform/platform-api/src/com/intellij/openapi/ui/DialogWrapper.java]: Updated accordingly

File: platform/platform-api/src/com/intellij/openapi/ui/DialogWrapper.java (hunk 1 above)
Comment (jgaillard85) [reply platform/platform-api/src/com/intellij/openapi/ui/DialogWrapper.java]: Removeed

**Comment by dahlstrom-g** [comment]
@kkmlc I'm not seeing this change in JetBrains:master where the update above shows it merged. Do you know why?

**Comment by kkmic** [comment]
'Cause I did it a wrong way. Let me do it the right way. And it'll be there. Sorry, doing it the first time.

**Comment by dahlstrom-g** [comment]
I see it commit d796888 on master now. Thanks!

**Pull request merged**